src/nccl_ofi_rdma.c

```diff
@@ -7479,7 +7479,14 @@ nccl_net_ofi_rdma_device_release(nccl_net_ofi_device_t *base_device)
 
 	unsigned num_domains = HASH_COUNT(device->base.domain_table);
 	if (num_domains > 0) {
-		NCCL_OFI_INFO(NCCL_NET, "%u domains still active at close", num_domains);
```

Comment:
This became useless because of the hack to get P5en launched, but if there's a domain with active endpoints, there absolutely should be an info here; that's a bug we should fix.